pom.xml

```diff
@@ -1,190 +1,503 @@
+        <project.report.sourceEncoding>UTF-8</project.report.sourceEncoding>
+
+        <maven.compiler.release>11</maven.compiler.release>
+
+        <!-- Official Maven Plugins -->
+        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+        <maven-war-plugin.version>3.3.1</maven-war-plugin.version>
+        <maven-dependency-plugin.version>3.1.2</maven-dependency-plugin.version>
+        <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+        <maven-failsafe-plugin.version>3.0.0-M5</maven-failsafe-plugin.version>
+        <maven-surefire-report-plugin.version>3.0.0-M5</maven-surefire-report-plugin.version>
+
+        <!-- Cargo maven plugin -->
+        <cargo-maven2-plugin.version>1.8.4</cargo-maven2-plugin.version>
+
+        <!-- Jakarta EE API -->
+        <jakartaee-api.version>8.0.0</jakartaee-api.version>
+
+        <!-- Arquillian BOM -->
+        <arquillian-bom.version>1.6.0.Final</arquillian-bom.version>
+        <!-- Test Utilities -->
+        <junit.version>4.13.1</junit.version>
+        <hamcrest.version>2.2</hamcrest.version>
+        <assertj.version>3.18.1</assertj.version>
+        <!-- By default, skip tests -->
+        <skipTests>true</skipTests>
+
+        <!-- Payara server -->
+        <payara.version>5.2020.7</payara.version>
+        <arquillian-payara.version>2.3.1</arquillian-payara.version>
+
+        <!-- Application properties -->
+        <webapp.databaseTempDir>/tmp</webapp.databaseTempDir>
+        <webapp.graphTraversalUrl>
+            http://localhost:8080/cargo-tracker/rest/graph-traversal/shortest-path
+        </webapp.graphTraversalUrl>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- Jakarta EE API -->
+            <dependency>
+                <groupId>jakarta.platform</groupId>
+                <artifactId>jakarta.jakartaee-api</artifactId>
+                <version>${jakartaee-api.version}</version>
+                <scope>provided</scope>
+            </dependency>
+
+            <!-- Arquillian Bill Of Management used to reduce collision between testing
+                dependencies. Using a dependency manager we don't need to specify the version
+                of each Arquillian dependency. -->
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>${arquillian-bom.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>${junit.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <!-- https://mvnrepository.com/artifact/org.hamcrest/hamcrest -->
+            <dependency>
+                <groupId>org.hamcrest</groupId>
```

Comment:
In my opinion would be better to use only JUnit and Arquillian for testing

Reply:
Is this dependency used? If it is used, it is fine. It is just a test utility.